src/backend/_scss/application.scss

```diff
@@ -2097,3 +2097,7 @@ a.skip:hover {
   margin-bottom: 5px;
   margin-left: 5px;
 }
+
+.center{
+  display: block;
```

Comment:
@agbilotia1998 the class name is `.center` but it is used for `display` property?